api/metrics/src/main/java/io/opentelemetry/api/metrics/DoubleCounterBound.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.metrics;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/** A {@code Bound Instrument} for a {@link DoubleCounter}. */
+@ThreadSafe
+public interface DoubleCounterBound extends BoundSynchronousInstrument {
```

Comment:
why the rename? I think `BoundDoubleCounter` was a better name.

Reply:
Was mostly trying to have classes together grouped by name. I understand your concern, will fix.

Reply:
Done.